todo-list/src/App.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div id="app">
+    <h1>ToDoリスト（VUE版）</h1>
+    <label><input type="radio" v-model="state" value="all" />すべて</label>
+    <label><input type="radio" v-model="state" value="active" />作業中</label>
+    <label><input type="radio" v-model="state" value="completed" />完了</label>
+
+    <list
+      :todos="filterTodos"
+      @remove="removeTask"
+      @change="changeState"
+    ></list>
+
+    <h2>新規タスクの追加</h2>
+    <input v-model="task" size="20" /><button class="button" @click="addTask">
+      追加
+    </button>
+  </div>
+</template>
+
+<script>
+import list from "@/components/list.vue";
+
+export default {
+  name: "app",
+  components: {
+    list,
+  },
+  data() {
+    return {
+      state: "all",
+      task: "",
+      todos: [],
+    };
+  },
+  methods: {
+    //新規タスクの追加
+    addTask: function () {
```

Comment:
これでも良いのですが、以下のように省略して記述することが多いです。
```suggestion
    addTask() {
```

Reply:
了解しました。修正しました。
以降、省略記述にするように致します<m(__)m>